api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -93,6 +95,14 @@ void createLineageEvent(
       PGobject event,
       String producer);
 
+  @SqlUpdate(
+      "INSERT INTO lineage_events ("
+          + "event_time, "
+          + "event, "
+          + "producer) "
+          + "VALUES (?, ?, ?)")
+  void createLineageEvent(Instant eventTime, PGobject event, String producer);
```

Comment:
Minor: Although the params are generic, this DAO method would only be used for inserting `DatasetEvent`s. I think for clarity, I would use `createDatasetEvent()`. Also, do the remaining columns in the `lineage_events` default to `null`?